### PR TITLE
Fixes for issues Tom F sees when running the ATH locally

### PR DESCRIPTION
**Pull request by tfennelly**
# Description

Depends on ...

* https://github.com/jenkinsci/js-modules/pull/4
* https://github.com/jenkinsci/js-builder/pull/10
* https://github.com/jenkinsci/sse-gateway-plugin/pull/10
* https://github.com/jenkinsci/blueocean-plugin/pull/491

@jenkinsci/code-reviewers @reviewbybees 

- [x] Remove breadcrumb bar on classic jenkins page because it's blocking some interactions because of it's sticky positioning at the top of the page.
- [x] Get the ATH working.
- [x] Seeing some SSE errors locally .... this might be good as they might be the same issues as we're seeing on the CI server.

Yo @scherler and @michaelneale ... fyi

### CODE REVIEW

**Comment by michaelneale** [comment]
@tfennely cool. Got any ideas on those SSE problems with it? Thor had issues with bumping the client to 0.0.8 the other day IIRC

**Comment by tfennelly** [comment]
@michaelneale the SSE issue would be that 0.0.8 requires a new release of the Gateway plugin itself, which I forgot to do but will not do now until I've addressed some of the possible locking issues.

As for the ATH and nodegit .... I ran into those issues and fixed them on my machine by deleting all global NPM packages (installed with an older npm). After doing that ... no more nodegit errors for me ..... a new set of errors now :)

**Comment by tfennelly** [comment]
I now have a locally working ATH with up to date SSE etc. Now I can start working on the SSE issues that I started this for. Hmmm ..... maybe I should go merging this work first and come back with another PR for the SSE specific fixes ([JENKINS-37921](https://issues.jenkins-ci.org/browse/JENKINS-37921) etc).

File: nightwatch.conf.js [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
-        throw 'Jenkins not running. Failed to find file: ' + jenkins_url_file;
-    }
+    if (process.env.LAUNCH_URL) {
+        //
+        // This allows you to run the tests against a Jenkins instance of
+        // your choosing when manually running nightwatch e.g. one running in your
+        // IDE, allowing you to debug the server etc.
+        //
+        // Command line example:
+        //   LAUNCH_URL=http://localhost:8080/jenkins nightwatch
```
Comment (scherler) [inline nightwatch.conf.js]: Awesome!!!

File: package.json [hunk 2]
```diff
@@ -4,7 +4,9 @@
   "description": "Blue Ocean acceptance tests",
   "main": "index.js",
   "scripts": {
-    "test": "nightwatch"
+    "test": "npm run nightwatch",
+    "nightwatch": "PATH=./node:./node/npm:$PATH nightwatch",
```
Comment (scherler) [inline package.json]: Add to the path:

./node_modules/.bin

So we are sure nightwatch it's taken from there

**Comment by scherler** [comment]
:bee:

File: package.json (hunk 2 above)
Comment (tfennelly) [inline package.json]: @scherler Ahh ... good idea buddy !!

Comment (tfennelly) [inline package.json]: Oh actually @scherler .... no need to add it since it's already in there automatically when running `npm run`.

Comment (scherler) [inline package.json]: @tfennelly right

**Comment by scherler** [comment]
@tfennelly a general observation: 

```
//// FIXME: Disabled because of https://issues.jenkins-ci.org/browse/JENKINS-37843
￼//// to enable remove ! before function
```

to skip a test you do not have to comment with ```//``` but add ! before the function. as in:

```
"bla": !function....
```

Further commenting those tests are not the solution, further they do work for me. I will try after lunch my other laptop, while I am waiting for vivek to start work ;)

**Comment by tfennelly** [comment]
@scherler I've tried those tests on 2 different machines and they failed on both, so imo there's def something wrong with them. If they run for you then there's no point in you declaring them as working again and re-enabling them.

As for comments Vs using a sneaky little bang character ... I'm preferring the more obvious and uncommenting approach.

**Pull request merged**